AnnoDesigner/Localization/TranslateExtension.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Windows.Data;
+
+namespace AnnoDesigner.Localization
+{
+    public class TranslateKeyConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            var translations = value as IDictionary<string, string>;
+            var key = parameter as string;
+
+            if (translations.ContainsKey(key))
+            {
+                return translations[key];
+            }
+            return $"!{key}";
+        }
+
+        public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            throw new NotImplementedException();
+        }
+    }
+
+    public class Translate : Binding
+    {
+        private static TranslateKeyConverter TranslateConverter { get; } = new TranslateKeyConverter();
+
+        public string Key
+        {
+            set
+            {
+                ConverterParameter = value;
+            }
+        }
+
+        public Translate() : base("InstanceTranslations")
```

Comment:
Adding the initial binding path `InstanceTranslations` doesn't do anything, and is a little confusing to someone not familiar with what the parameter represents - its harmless, but by extension of that fact, isn't needed either.

Also, as we discussed on discord, are we renaming this class to `Localize` rather than `Translate`?

Reply:
The initial path is necessary for this to work. Translated key is not stored in Path but in ConverterParameter.

I'll rename this class to Localize.